alphastats/DataSet.py

```diff
@@ -64,19 +64,20 @@ def __init__(
         """
         self._check_loader(loader=loader)
 
+        self._data_harmonizer = DataHarmonizer(loader, sample_column)
```

Comment:
I would be in favor of moving the data harmonization out of dataset and insert it as last step in the loaders instead. This way we could immediately pass the harmonized data as rawinput and remove the loader from the arguments. The loader could then be a transient object, removing one instance of data duplication.

Reply:
agree, still I followed the "dont touch the loaders for now" principle :-)